blocks/press-cards/press-cards.css

```diff
@@ -96,10 +96,14 @@
         display: inline-block;
         height: auto;
         align-items: flex-end;
-        min-height: 20rem;
+        width: 15rem;
```

Comment:
should the height also be set to prevent misalignment of cards like this? 
![image](https://github.com/hlxsites/zeiss/assets/1191451/62d45657-0aaf-4916-9444-46285609531d)
The assumption is that the images larger than the parent div would get cropped and smaller than the parent div would get scaled, maintaining the aspect-ratio all the times.